api/web3server.go

```diff
@@ -387,10 +389,15 @@ func (svr *web3Handler) call(in *gjson.Result) (interface{}, error) {
 		return "0x" + ret, nil
 	}
 	exec, _ := action.NewExecution(to, 0, value, gasLimit, big.NewInt(0), data)
-	ret, _, err := svr.coreService.ReadContract(context.Background(), callerAddr, exec)
+	ret, receipt, err := svr.coreService.ReadContract(context.Background(), callerAddr, exec)
 	if err != nil {
 		return nil, err
 	}
+
+	if len(receipt.ExecutionRevertMsg) > 0 {
+		return "0x" + ret, status.Error(codes.InvalidArgument, receipt.ExecutionRevertMsg)
```

Comment:
please verify that you get the expected API response
```
{
    "jsonrpc":"2.0",
    "id":1,
    "error":{
         "code":3,
         "message":"execution reverted: require test",
         "data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000c7265717569726520746573740000000000000000000000000000000000000000"
    }
}
```

Reply:
> please verify that you get the expected API response
> 
> ```
> {
>     "jsonrpc":"2.0",
>     "id":1,
>     "error":{
>          "code":3,
>          "message":"execution reverted: require test",
>          "data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000c7265717569726520746573740000000000000000000000000000000000000000"
>     }
> }
> ```
done, pls review

Reply:
`status` lib is used in grpc, not in http